hub/core/meta/tensor_meta.py

```diff
@@ -71,6 +71,16 @@ def create(
     def load(key: str, storage: StorageProvider):
         return TensorMeta(get_tensor_meta_key(key), storage)
 
+    @staticmethod
+    def copy(
+        key: str,
+        src_meta: Meta,
+        dst_storage: StorageProvider,
+    ):
+        src_dict = src_meta.to_dict()
+        new_tensor_meta = TensorMeta.create(key, dst_storage)
+        return new_tensor_meta.from_dict(src_dict)
+
```

Comment:
delete this, `Meta` should have the non-staticmethod version in my previous comment